pkg/apis/build/v1alpha1/build_source.go

```diff
@@ -4,15 +4,34 @@
 
 package v1alpha1
 
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
 // BuildSourceType enumerates build source type names.
 type BuildSourceType string
 
+// LocalCopy defines a source type that waits for user input, as in a local copy into a POD.
+const LocalCopy BuildSourceType = "LocalCopy"
+
 // BuildSource remote artifact definition, also known as "sources". Simple "name" and "url" pairs,
```

Comment:
doesn't this comment need to be updated to accont for both remote artifacts and local ?

Reply:
I've updated the comments on the latest commit. With the following:

```go
// LocalCopy represents a alternative build workflow that instead of `git clone` the repository, it
// employs the data uploaded by the user, streamed directly into the POD.
const LocalCopy BuildSourceType = "LocalCopy"

// HTTP defines a (HTTP) remote artifact, which will be downloaded into the build POD, right before
// the build process starts. Represents a remote dependency.
const HTTP BuildSourceType = "HTTP"
```

Reply:
works for me ... marking resolved